source/css/sass/mixins/_typography.scss

```
@@ -67,6 +67,25 @@
   vertical-align: middle;
 }
 
+@mixin font-variant-position($position) {
+  font-size: 75%;
+  line-height: 0;
+  position: relative;
+  vertical-align: baseline;
+
+  @if $position == super {
+    top: -0.5em;
+  } @else {
+    @include _error("Does not match '#{$position}'");
```

The error's for the person using the mixin, so `Does not match 'foo'` isn't particularly useful.

```suggestion
    @include _error("Unknown position '#{$position}'");
```